Example/UI Examples/CardFieldViewController.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  CardFieldViewController.swift
+//  UI Examples
+//
+//  Created by Ben Guo on 7/19/17.
+//  Copyright © 2017 Stripe. All rights reserved.
+//
+
+import UIKit
+import Stripe
+
+class CardFieldViewController: UIViewController {
+
+    let cardField = STPPaymentCardTextField()
```

Comment:
I thought about demoing a few different styles a la https://stripe.com/docs/elements/examples , but couldn't think of enough interesting variations...